lesson_15/class_rhombus.py

```diff
@@ -0,0 +1,35 @@
+class Rhombus:
+
+    side_a: int
```

Comment:
It is better to make such type of attributes private and provide getter methods for them.